File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/Machines.java
package com.pennapps.labs.pennmobile.classes;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Machines {

    @SerializedName("dryers")
    @Expose
    private MachineList dryers;
    @SerializedName("washers")
    @Expose
    private MachineList washers;
    @SerializedName("details")
    @Expose
    private List<MachineDetail> machineDetailList;

    public MachineList getDryers() {
        return dryers;
    }

    public MachineList getWashers() {
        return washers;
    }

    public List<MachineDetail> getMachineDetailList() {
        return machineDetailList;
    }
}

File: PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/Machines.kt
package com.pennapps.labs.pennmobile.classes

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Machines {
    @SerializedName("dryers")
    @Expose
    val dryers: MachineList? = null

    @SerializedName("washers")
    @Expose
    val washers: MachineList? = null

    @SerializedName("details")
    @Expose
    val machineDetailList: List<MachineDetail>? = null
}
